src/main/resources/application.properties

```diff
@@ -3,6 +3,52 @@ DOMAIN=localdomain
 OAUTH_CLIENT_ID=null
 OAUTH_SECRET=null
 
+SKIP_SSL_VALIDATION="true"
```

Comment:
I don't see where this is used. Can it be removed? Also, why is it a string instead of a boolean

Reply:
Yes; it's a remnant from the sample application that I didn't end up using.